app/client/keybase/keyvault.go

```diff
@@ -0,0 +1,296 @@
+package keybase
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/vault/api"
+	"github.com/pokt-network/pocket/shared/crypto"
+)
+
+// VaultKeybase implements the Keybase interface using HashiCorp Vault
+type VaultKeybase struct {
```

Comment:
Let's lowercase it if we don't need to expose `VaultKeybase` outside of this package.